docs/usage/minimum-release-age.md

```diff
@@ -0,0 +1,374 @@
+            "datasource": "docker",
+            "depType": "stage",
+            "updates": [
+              {
+                "bucket": "major",
+                "newVersion": "11.40.5",
+                "newValue": "11.40.5",
+                "newMajor": 11,
+                "newMinor": 40,
+                "newPatch": 5,
+                "updateType": "major",
+                "isBreaking": true,
+                "newDigest": "sha256:81bbc8c8c561f6c4c2d059a5bcdfc95ef837682a41ac45bfbc1380d8d07dc941",
+                "branchName": "renovate/main-ghcr.io-renovatebot-base-image-11.x"
+              }
+            ],
+          }
+      // ...
+    ],
+    "github-actions": [
+      {
+        "deps": [
+          // NOTE that we do get a release timestamp for this GitHub Action major version bump but we do for this
+          {
+            "depName": "actions/setup-node",
+            "commitMessageTopic": "{{{depName}}} action",
+            "datasource": "github-tags",
+            "versioning": "docker",
+            "depType": "action",
+            "replaceString": "actions/setup-node@49933ea5288caeca8642d1e84afbd3f7d6820020 # v4.4.0",
+            "autoReplaceStringTemplate": "{{depName}}@{{#if newDigest}}{{newDigest}}{{#if newValue}} # {{newValue}}{{/if}}{{/if}}{{#unless newDigest}}{{newValue}}{{/unless}}",
+            "currentValue": "v4.4.0",
+            "currentDigest": "49933ea5288caeca8642d1e84afbd3f7d6820020",
+            "updates": [
+              {
+                "bucket": "major",
+                "newVersion": "v6.0.0",
+                "newValue": "v6.0.0",
+                "newDigest": "2028fbc5c25fe9cf00d9f06a71cc4710d4507903",
+                "releaseTimestamp": "2025-10-14T02:37:06.000Z",
+                "newVersionAgeInDays": 10,
+                "newMajor": 6,
+                "newMinor": 0,
+                "newPatch": 0,
+                "updateType": "major",
+                "isBreaking": true,
+                "libYears": 0.5323368531202435,
+                "branchName": "renovate/main-actions-setup-node-6.x"
+              }
+            ],
+            "packageName": "actions/setup-node",
+            "warnings": [],
+            "sourceUrl": "https://github.com/actions/setup-node",
+            "registryUrl": "https://github.com",
+            "mostRecentTimestamp": "2025-10-14T02:37:06.000Z",
+            "isAbandoned": false,
+            "currentVersion": "v4.4.0",
+            "currentVersionTimestamp": "2025-04-02T19:20:51.000Z",
+            "currentVersionAgeInDays": 204,
+            "isSingleVersion": true,
+            "fixedVersion": "v4.4.0"
+          },
+```
+
+</details>
+
+Given a log line such as ☝️ you can also use the following `jq` query to identify any dependencies (or their updates) that are missing the `currentVersionTimestamp` or `releaseTimestamp` fields like so:
```

Comment:
The query seems like an extra addition which is not needed here.

Reply:
We should inform them about relevant fields. How to extract them should be left to the user.

Reply:
I think it is fine to give examples for what people to run, but a table with the fields and there purpose would be helpful here

Reply:
To confirm, do we want to document i.e. what `missingCurrentVersionTimestamps` indicates, or down to the level of `bucket` and `registryUrls` etc? Just thinking how in-depth to focus